Beecrowd.Beginner.Solutions/1036-BashkarasFormula.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Beecrowd
+{
+    class BhaskarasFormula
+    {
+        public static void Run()
```

Comment:
Tests are missing.

Reply:
How Can I write a test that expects multi-line results?

Reply:
You can use the `Theory` instead of `Fact`. If you look at the other tests you can easily figure it out.

Reply:
The other tests are for Multi line inputs, I want to test multi-line output

Reply:
In this case, probably you can use `Assert.Collection` to verify a list of results. for reference you can look at this link: https://bartwullems.blogspot.com/2020/03/xunit-assertcollection.html